common/src/main/scala/common/validation/Validation.scala

```diff
@@ -41,7 +42,11 @@ object Validation {
   implicit class ValidationTry[A](val e: ErrorOr[A]) extends AnyVal {
     def toTry: Try[A] = e match {
       case Valid(options) => Success(options)
-      case Invalid(err) => Failure(new RuntimeException(s"Error(s): ${err.toList.mkString(",")}"))
+      case Invalid(err) => Failure(AggregatedMessageException("Error(s)", err.toList))
+    }
+    def toTry(context: String): Try[A] = e match {
+      case Valid(options) => Success(options)
+      case Invalid(err) => Failure(AggregatedMessageException(context, err.toList))
```

Comment:
The codecov browser plugin says these `cromwell-common` lines aren't covered, but I'll put it back to 100% in another commit. ;)